src/sql/workbench/browser/designer/designer.ts

```diff
@@ -719,6 +724,24 @@ export class Designer extends Disposable implements IThemable {
 				});
 				component = dropdown;
 				break;
+			case 'editableDropdown':
```

Comment:
you only implemented the component creation, you have to also implement the set component value for the new component type. please do an end to end testing.

Reply:
As mentioned in the PR, this initial ones is for moving files and not table designer part.